src/gwt/src/org/rstudio/core/client/widget/GridViewerFrame.java

```diff
@@ -50,4 +62,17 @@ private final native void setDataNative(WindowEx frameContentWindow, JavaScriptO
    private final native void setOptionNative(WindowEx frameContentWindow, String option, String value) /*-{
       frameContentWindow.setOption(option, value);
    }-*/;
+   
+   private final native void setColumnDefinitionsUIVisibleNative(WindowEx frameContentWindow, boolean value, Operation onColumnDefsChange) /*-{
```

Comment:
We generally try to keep code to 80 columns wide -- unfortunately there's no way to make Eclipse do this automatically, but you can make it show a print margin at 80 columns so you know where to hard-wrap.